yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -10,29 +10,39 @@ import { makeAppendOnlyTreeSnapshot, makeHeader } from './l2_block_code_to_purge
  * The data that makes up the rollup proof, with encoder decoder functions.
  */
 export class L2Block {
+  #l1BlockNumber?: bigint;
+
   constructor(
     /** Snapshot of archive tree after the block is applied. */
     public archive: AppendOnlyTreeSnapshot,
     /** L2 block header. */
     public header: Header,
     /** L2 block body. */
     public body: Body,
-  ) {}
+    /** Associated L1 block num */
+    l1BlockNumber?: bigint,
```

Comment:
I am retarded and I probably got this change here when I was doing freestyle with branch re-ordering. Since this PR got accidentally merged I am reverting this change [here](https://github.com/AztecProtocol/aztec-packages/pull/5371).